Big-Spender/readme.md

```diff
@@ -112,7 +122,9 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE: SELECT SUM(amount) AS total_amount
+FROM spends
+WHERE date = '2021-03-01' OR date = '2021-04-01'
```

Comment:
yep, this is good. Could we use IN instead? Or what other ways would this be possible?